pyproject.toml

```diff
@@ -1,9 +1,9 @@
 [project]
 name = "dark-matter"
-version = "5.1.2"
+version = "6.0.0"
 description = "Python library and utility scripts for working with genetic sequence data."
 readme = "README.md"
-requires-python = ">=3.10"
+requires-python = ">=3.10,<3.14"
```

Comment:
Bump to `>=3.12`?
Why `<3.14`?

Reply:
Unfortunately, 3.13 doesn't even work (we have a dependency on ete3 which breaks under 3.13) so it should in fact be `<3.13`. I'm going to address this in another pr by testing `ete4` and also seeing how far I can get under `3.14`. I don't know how (or if it's possible) to declare valid version ranges depending on what you need.